diff --git a/build/media-suite_deps.sh b/build/media-suite_deps.sh
index 0fb93da..8c25ec7 100644
--- a/build/media-suite_deps.sh
+++ b/build/media-suite_deps.sh
@@ -119,7 +119,7 @@ SOURCE_REPO_XAVS2=https://github.com/pkuvcl/xavs2.git
 SOURCE_REPO_XAVS=https://github.com/Distrotech/xavs.git
 SOURCE_REPO_XEVD=https://github.com/mpeg5/xevd.git
 SOURCE_REPO_XEVE=https://github.com/mpeg5/xeve.git
-SOURCE_REPO_XVC=https://github.com/divideon/xvc.git
+SOURCE_REPO_XVC=https://github.com/galcius/xvc.git
 SOURCE_REPO_XVID=https://github.com/m-ab-s/xvid.git
 SOURCE_REPO_ZIMG=https://github.com/sekrit-twc/zimg.git
 SOURCE_REPO_ZLIB=https://github.com/madler/zlib.git
